src/components/block-content.tsx

```diff
@@ -35,6 +35,31 @@ InternalLink.propTypes = {
   children: PropTypes.node.isRequired
 };
 
+type ExternalLinkProps = {
+  mark: {
+    _key: string;
+    _type: string;
+    blank: boolean;
+    href: string;
+  };
+  children: ReactNode;
+};
+
+const ExternalLink: FC<ExternalLinkProps> = ({children, mark}) => {
+  const reference = {...mark, children: children[0], isInternal: false};
```

Comment:
```suggestion
  const reference = {...mark, children, isInternal: false};
```

Is it ok to just pass `children` through? Unsure why we need to access the first element? 🤔

Reply:
Seems this is true!